<launch>
    <arg name="use_fake_objects" default="false" doc="Return fake object locations" />
    <arg name="cont_requests" default="false" doc="Continuously request objects" />
    <arg name="dope_config" default="$(find triple_s_util)/config/dope.yaml" doc="DOPE configuration" />
    <arg name="bin_picking_config" default="$(find triple_s_util)/config/bin_picking.yaml" doc="Bin picking configuration" />

    <rosparam command="load" file="$(arg dope_config)" ns="/dope" />
    <rosparam command="load" file="$(arg bin_picking_config)" ns="/bin_picking" />

    <node name="find_object" pkg="triple_s_util" type="find_object.py" output="screen" unless="$(arg use_fake_objects)" />
    <node name="find_object" pkg="triple_s_util" type="fake_find_object.py" output="screen" if="$(arg use_fake_objects)" />

    <node name="bin_picking_sequencer" pkg="triple_s_util" type="bin_picking_sequencer.py" output="screen" />

    <node name="continous_requests" pkg="triple_s_util" type="continuous_random.py" output="screen" if="$(arg cont_requests)" />
</launch>